HW15/Task_6.py

```diff
@@ -0,0 +1,44 @@
+
+import os
+import logging
+from collections import namedtuple
+import argparse
+
+
+FORMAT = '{msg}'
+
+logging.basicConfig(filename='log1.txt', format=FORMAT, style='{', level=logging.INFO, encoding='utf-8')
+logger = logging.getLogger(__name__)
+Object_dir = namedtuple('Object', ['name', 'exemp', 'is_dir', 'path'])
+
+
+
+def dir_write(dir_=os.getcwd()):
+    os.chdir(dir_)
+    # print(dir_)
+    list_obj = []
+    for obj in os.listdir(os.getcwd()):
```

Comment:
Так если передана конкретная директория, то может сделать проще
```suggestion
    # print(dir_)
    list_obj = []
    for obj in os.listdir(dir_):
```